scripts/cleanup.py

```diff
@@ -200,7 +201,33 @@ def main():
         help="Root path of rageshakes (eg /home/rageshakes/bugs/)",
     )
 
+    parser.add_argument(
+        "--repeat-delay-hours",
+        dest="repeat_delay_hours",
+        type=float,
+        help="After each execution, wait this number of hours then run again. An alternative to configuring a cronjob for ongoing cleanup.",
+    )
+
     args = parser.parse_args()
+
+    if args.repeat_delay_hours:
```

Comment:
to distinguish from `--repeat-delay-hours=0`:

```suggestion
    if args.repeat_delay_hours is defined:
```